src/main/java/net/minestom/server/ServerProcessImpl.java

```diff
@@ -139,7 +137,7 @@ public ServerProcessImpl() {
 
         this.server = new Server(packetParser);
 
-        this.dispatcher = ThreadDispatcher.of(ThreadProvider.counter(), ServerFlag.DISPATCHER_THREADS);
+        this.dispatcher = ThreadDispatcher.serverProcess();
```

Comment:
Not a fan of this name, probably the logic of choosing from inside #serverProcess should just be here instead